source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -644,14 +644,14 @@ export abstract class CardElement {
 }
 
 export class TextBlock extends CardElement {
-    private _computedLineHeight: number;
+    private lineHeight: number;
```

Comment:
All private fields should start with _

Why change the name anyway? Can we please keep _computedLineHeight?